src/main.py

```diff
@@ -1 +1,6 @@
+# python3.7.5
+
+import os
+
 print('hello world yashio!')
+
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
 W391 blank line at end of file